devicetypes/smartthings/zooz-4-in-1-sensor.src/zooz-4-in-1-sensor.groovy

```diff
@@ -236,3 +282,63 @@ private secure(physicalgraph.zwave.Command cmd) {
 private secureSequence(commands, delay = 200) {
 	delayBetween(commands.collect{ secure(it) }, delay)
 }
+
+def getConfigurationCommands() {
+	log.debug "getConfigurationCommands"
+	def result = []
+	if (isEverspringSP815()) {
+		if (!state.temperatureAndHumidityReport) state.temperatureAndHumidityReport = getEverspringDefaults[1]
+		if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = getEverspringDefaults[2]
+		Integer temperatureAndHumidityReport = (settings.temperatureAndHumidityReport as Integer) ?: everspringDefaults[1]
+		Integer retriggerIntervalSettings = (settings.retriggerIntervalSettings as Integer) ?: everspringDefaults[2]
+		if (!isConfigured() || (temperatureAndHumidityReport != state.temperatureAndHumidityReport || retriggerIntervalSettings != state.retriggerIntervalSettings)) {
```

Comment:
I want to recommend some formatting cleanup so that this function is easier read.

- Put a blank line above this if statement, as well as the `if (isEverspringSP815()) {` line.
- Move the variable declaration to the top of the if block.
- Break these to lines:
```
if (!state.temperatureAndHumidityReport) state.temperatureAndHumidityReport = getEverspringDefaults[1]
if (!state.retriggerIntervalSettings) state.retriggerIntervalSettings = getEverspringDefaults[2]
```
into
```
if (!state.temperatureAndHumidityReport) {
	state.temperatureAndHumidityReport = getEverspringDefaults[1]
}
if (!state.retriggerIntervalSettings) {
	state.retriggerIntervalSettings = getEverspringDefaults[2]
}
```
So that you'd have:
```
variable declarations

if {
}
if {
}

if{
```
- And I'd put a blank line after `state.configured = false` and before the if blocks

Reply:
right, should be fine now